testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication.py

```diff
@@ -173,15 +173,17 @@ def test_replication_configuration_with_pull_replication(params_from_base_test_s
     # One shot replication
     sg_added_doc_ids, cbl_added_doc_ids, session = setup_sg_cbl_docs(params_from_base_test_setup, sg_db=sg_db, base_url=base_url, db=db,
                                                                      cbl_db=cbl_db, sg_url=sg_url, sg_admin_url=sg_admin_url, sg_blip_url=sg_blip_url,
-                                                                     replication_type="pull", channels=channels, replicator_authenticator_type=authenticator_type)
+                                                                     replication_type="pull", channels=channels, replicator_authenticator_type=authenticator_type,
+                                                                     attachments_generator=attachments_generator)
     sg_docs = sg_client.get_all_docs(url=sg_admin_url, db=sg_db)
     if sg_mode == "di":
         cookie, session = sg_client.create_session(sg_admin_url, sg_db, "autotest")
         authenticator = Authenticator(base_url)
         replicator_authenticator = authenticator.authentication(session, cookie, authentication_type="session")
         replicator = Replication(base_url)
-        replicator.configure_and_replicate(cbl_db, replicator_authenticator, target_url=sg_blip_url, continuous=False,
+        replicator.configure_and_replicate(cbl_db, replicator_authenticator, target_url=sg_blip_url, continuous=True,
```

Comment:
why the change from one-shot to continuous replication?

Reply:
changed it back to False